docs/quickstart.rst

```diff
@@ -134,7 +134,7 @@ execution to available endpoints.  For most use-cases that will use the
 
 .. note::
     Like most FaaS platforms, the function must be registered with the upstream
-    web services before it can be executed on a remote endopint.  While one can
+    web services before it can be executed on a remote endpoint.  While one can
```

Comment:
Admittedly, it's too bad we don't have an endopint.  Whatever it is, it's probably delicious.  "Mmm, beer."